kafka-client-examples/e2e-test/src/main/java/dev/responsive/examples/e2etest/E2ETestDriver.java

```diff
@@ -243,12 +246,20 @@ private void pollOnce() {
       }
       consumedOutputOffsets.put(cr.partition(), cr.offset());
       recordsProcessed += poppedOffsets.size();
+      maybeFireSetupCompleteSignal();
       maybeLogConsumed();
       maybeLogAllConsumed();
       cs.updateReceived(poppedOffsets, ps.partition(), cr.value().digest());
     }
   }
 
+  private void maybeFireSetupCompleteSignal() {
+    if (!setupCompleteSignalFired && recordsProcessed > 0) {
+      LOG.info("Received at least one output record, setup is complete");
+      Lifecycle.setupComplete(null);
```

Comment:
what's `null` here? can we give antithesis more info? same with most other calls to the antithesis SDK, seems like we're passing in lots of `null`

Reply:
I'm going to merge this for now to unblock the nightlies but I'll follow up on this